jwst/extract_1d/extract_1d_step.py

```diff
@@ -137,8 +145,10 @@ class Extract1dStep(Step):
     log_increment = integer(default=50)  # increment for multi-integration log messages
     subtract_background = boolean(default=None)  # subtract background?
     use_source_posn = boolean(default=None)  # use source coords to center extractions?
-    center_xy = int_list(min=2, max=2, default=None)  # IFU extraction x/y center
+    center_xy = float_list(min=2, max=2, default=None)  # IFU extraction x/y center
     apply_apcorr = boolean(default=True)  # apply aperture corrections?
+    ifu_autocen = boolean(default=False) # Auto source centering for IFU point source data.
+    ifu_rfcorr = boolean(default=False) # Apply 1d residual fringe correction
     soss_atoca = boolean(default=True)  # use ATOCA algorithm
```

Comment:
I'm a little hesitant here because I'm not sure about all the ways Extract1D can be calling models vs model containers, and if I add something it may cause an issue for some obscure spec mode I'm not familiar with.  Do you think the check on instr=MIRI before actually running the fringe code is enough?

Reply:
Yes I think adding the instr=MIRI is good enough